src/t8_cmesh/t8_cmesh_cxx.cxx

```diff
@@ -56,133 +56,134 @@ t8_cmesh_uniform_bounds (t8_cmesh_t cmesh, int level,
 
-  if (cmesh->num_trees_per_eclass[T8_ECLASS_PYRAMID] == 0) {
-    t8_gloidx_t         global_num_children;
-    t8_gloidx_t         first_global_child;
-    t8_gloidx_t         child_in_tree_begin_temp;
-    t8_gloidx_t         last_global_child;
-    t8_gloidx_t         children_per_tree = 0;
-    t8_gloidx_t         first_class_children_per_tree = -1;
+  t8_gloidx_t         global_num_children;
+  t8_gloidx_t         first_global_child;
+  t8_gloidx_t         child_in_tree_begin_temp;
+  t8_gloidx_t         last_global_child;
+  t8_gloidx_t         children_per_tree = 0;
+  t8_gloidx_t         first_class_children_per_tree = -1;
 #ifdef T8_ENABLE_DEBUG
-    t8_gloidx_t         prev_last_tree = -1;
+  t8_gloidx_t         prev_last_tree = -1;
 #endif
-    int                 tree_class;
-    t8_eclass_scheme_c *tree_scheme;
-
-    /* Compute the number of children on level in each tree */
-    for (tree_class = T8_ECLASS_ZERO; tree_class < T8_ECLASS_COUNT;
-         ++tree_class) {
-      /* We iterate over each element class and get the number of children for this
-       * tree class.
-       * Currently we do not supported different numbers of children for different classes.
-       * Thus, if we encounter this situation, we abort with an error.
-       * Different numbers of children for different classes will be supported in the future.
-       */
-      if (cmesh->num_trees_per_eclass[tree_class] > 0) {
-        tree_scheme = ts->eclass_schemes[tree_class];
-        T8_ASSERT (tree_scheme != NULL);
-        children_per_tree =
-          tree_scheme->t8_element_count_leafs_from_root (level);
-        if (first_class_children_per_tree >= 0
-            && first_class_children_per_tree != children_per_tree) {
-          SC_ABORT
-            ("Currently t8code does not support different leaf counts per tree.");
-        }
-        first_class_children_per_tree = children_per_tree;
+  int                 tree_class;
+  t8_eclass_scheme_c *tree_scheme;
+
+  /* Compute the number of children on level in each tree */
+  global_num_children = 0;
+  for (tree_class = T8_ECLASS_ZERO; tree_class < T8_ECLASS_COUNT;
+       ++tree_class) {
+    /* We iterate over each element class and get the number of children for this
+     * tree class.
+     * Currently we do not supported different numbers of children for different classes.
+     * Thus, if we encounter this situation, we abort with an error.
+     * Different numbers of children for different classes will be supported in the future.
+     */
+    if (cmesh->num_trees_per_eclass[tree_class] > 0) {
+      tree_scheme = ts->eclass_schemes[tree_class];
+      T8_ASSERT (tree_scheme != NULL);
+      children_per_tree =
+        tree_scheme->t8_element_count_leafs_from_root (level);
+      if (first_class_children_per_tree >= 0
+          && first_class_children_per_tree != children_per_tree) {
+        /*SC_ABORT
+           ("Currently t8code does not support different leaf counts per tree."); */
```

Comment:
The abort is commented out leaving the if brackets empty.
Pleae clean up.

Reply:
Replaced it with an Assertion